src/ansible_navigator/actions/settings.py

```diff
@@ -0,0 +1,201 @@
+"""The ``settings`` subcommand action."""
+
+import curses
+
+from typing import Any
+from typing import Dict
+from typing import List
+from typing import Tuple
+
+from .._yaml import human_dump
+from ..action_base import ActionBase
+from ..action_defs import RunStdoutReturn
+from ..app_public import AppPublic
+from ..configuration_subsystem import transform_settings
+from ..steps import Step
+from ..ui_framework import CursesLinePart
+from ..ui_framework import CursesLines
+from ..ui_framework import Interaction
+from . import _actions as actions
+from . import run_action
+
+
+def filter_content_keys(obj: Dict[Any, Any]) -> Dict[Any, Any]:
+    """Filter out some keys when showing content.
+
+    :param obj: The object to be filtered
+    :return: The object with keys filtered out
+    """
+    return {k: v for k, v in obj.items() if not k.startswith("__")}
+
+
+def color_menu(colno: int, colname: str, entry: Dict[str, Any]) -> Tuple[int, int]:
+    # pylint: disable=unused-argument
+    """Color the menu.
+
+    :param colno: Column number
+    :param colname: Column name
+    :param entry: Column value
+    :return: Tuple int used to color the menu
+    """
+    if entry["is_default"]:
+        return 2, 0
+    return 3, 0
+
+
+def content_heading(obj: Any, screen_w: int) -> CursesLines:
```

Comment:
Also, can `obj` be named and typed a bit more accurately? Is it a `Mapping[str, Any]`?

Reply:
+1 that we shouldn't use `Any` (ever). But dropping it here is a bigger change and will likely need some refactoring in the UI subsystem; it's probably best done in another PR.

Reply:
I think in this case we might be able to pull it off, since the content here is something we created and know the exact shape of........  will work with @alisonlhart and @thedoubl3j to test.

Reply:
This comment is outdated as settings has been refractored and obj is now of the class PresentableSettingsEntry.

Reply:
This comment is outdated as settings has been refractored and obj is now of the class PresentableSettingsEntry.